client/cmd/dexc/log.go

```diff
@@ -47,6 +47,20 @@ func initLogging(lvl string, utc bool) *dex.LoggerMaker {
 		fmt.Fprintf(os.Stderr, "failed to create custom logger: %v\n", err)
 		os.Exit(1)
 	}
+	var (
+		lvls      string
+		delimiter string
+	)
+	for k, v := range defaultLogLevelMap {
+		lvls = fmt.Sprintf("%s%s%s=%s", lvls, delimiter, k, v)
+		if delimiter == "" {
+			delimiter = ","
+		}
+	}
```

Comment:
This works, but you should feel free to add a method to the `LoggerMaker` so you can just pass the `defaultLogLevelMap` too..

Reply:
Ok.

Reply:
Yes, this makes more sense.